docs/README.md

```diff
@@ -70,10 +70,7 @@
   * [Encrypted streams](SodiumPlus/encrypted-streams.md)
     * [crypto_secretstream_xchacha20poly1305_init_push](SodiumPlus/encrypted-streams.md#crypto_secretstream_xchacha20poly1305_init_push)
     * [crypto_secretstream_xchacha20poly1305_init_pull](SodiumPlus/encrypted-streams.md#crypto_secretstream_xchacha20poly1305_init_pull)
-    * [crypto_secretstream_xchacha20poly1305_push](SodiumPlus/encrypted-streams.md#crypto_secretstream_xchacha20poly1305_push)
-    * [crypto_secretstream_xchacha20poly1305_pull](SodiumPlus/encrypted-streams.md#crypto_secretstream_xchacha20poly1305_pull)
```

Comment:
Please note that I am removing description of these methods since they are no longer directly usable as one needs to have access to the backend-specific state to use them which is now wrapped by the `init_*` method and is no longer accessible to the API users.